### PR TITLE
Fix/roe 2631 add trust error message

### PR DESCRIPTION
**Pull request by lduranteau**
### JIRA link
https://companieshouse.atlassian.net/browse/ROE-2631

### Change description
update of the list of allowed trust urls, including those containing transaction id and submission id

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/validation/stillInvolved.validation.ts [hunk 1]
```diff
@@ -68,8 +68,8 @@ export const checkTrustLegalEntityBeneficialOwnerStillInvolved = (appData: Appli
 
 export const checkTrustIndividualBeneficialOwnerStillInvolved = (appData: ApplicationData, req: Request): ValidationError[] => {
   const allowedUrls = [
-    [config.TRUST_ENTRY_URL, config.TRUST_INDIVIDUAL_BENEFICIAL_OWNER_URL],
-    [config.TRUST_ENTRY_WITH_PARAMS_URL, config.TRUST_INDIVIDUAL_BENEFICIAL_OWNER_URL],
+    [config.REGISTER_AN_OVERSEAS_ENTITY_URL, config.TRUSTS_URL, config.TRUST_INDIVIDUAL_BENEFICIAL_OWNER_URL],
```
Comment (markpit) [inline src/validation/stillInvolved.validation.ts:71]: is there a way of adding a unit test to cover the bug

Comment (lduranteau) [reply src/validation/stillInvolved.validation.ts:71]: I will look at creating a test that sends a request with a url containing transactionId and submissionId

Comment (lduranteau) [reply src/validation/stillInvolved.validation.ts:70]: Test added
Each test covers at least one validator

**Comment by markpit** [comment]
> I didn't realise that my editor had formatted the rest of certain files.

that's ok, it just looks like the "" quotation marks. We can leave it like this.

**Pull request merged**